src/actions/filters.js

```diff
@@ -0,0 +1,7 @@
+import actionTypes from '../constants/actions';
+// eslint-disable-next-line
+export const addFilter = ({ filterName, value }) => ({
```

Comment:
all actions use the key `data` to store the action data. Let's please use the same pattern here.